roles/example-cnf-app/tasks/trex/app.yaml

```diff
@@ -80,6 +81,10 @@
         - "reason==TestPassed"
         - "involvedObject.name={{ trex_app_cr_name }}"
     register: trex_result
+    retries: 60
```

Comment:
The previous task waits for `TestCompleted` event, which signifies the test has been completed. After test is completed, the `TestPassed` event is taking longer time? If you have the trex app cr describe, then it will list all the time taken for it to.

I would say delay with 5 sec and 5 retries is more than enough in case of delay in event processing. 60 is higher value which is not useful as the test is completed and increases the test time in case of failure.

Reply:
hi, @krsacme thank you for checking, changed that. I added a condition `until: trex_event.resources | length > 0` for the start event as well because the resources were empty as well.